napari/movie/naparimovie.py

```diff
@@ -0,0 +1,364 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+import imageio
+import copy
+
+from . import util
+
+
+class Movie:
+    def __init__(self, myviewer=None, inter_steps=15):
+
+        """Standard __init__ method.
+
+        Parameters
+        ----------
+        myviewer : napari viewer
```

Comment:
Can we go with the name `viewer` here instead of `myviewer`, it is more consistent with naming with the rest of the repo

Reply:
Sure I'll change it. As we are on this topic, I have to admit I'm not sure about the mechanism of the viewer key-binding functions. I think they all HAVE to have "viewer" as parameter, but I'm not sure which viewer it refers to. Currently I have this kind of definition ```def capture_keyframe_callback(self, viewer)```  where I'm not using ```viewer``` at all since I'm using ```self.myviewer```. Any recommendation on making this cleaner ?

Reply:
@kne42 can you answer this one

Reply:
@guiwitz The name of the parameter in `capture_keyframe_callback`  doesn't actually matter. It is bound to the viewer that calls `bind_key` on `capture_keyframe_callback`, as in `self.viewer.bind_key("f", capture_keyframe_callback)`. Since, as you said, `viewer` is not used, you can ignore it, but it is required to have that signature to be registered as a callback.

I think, in the future, we'll want to rework the keybinding system to be more general than just applying to viewers and layers so that it supports additional modes/plugins like yours in which it would make more sense :) I'll make an issue for it shortly.

Reply:
Thanks for the infos @kne42, I'm glad I didn't make a complete mess with these key-bindings!